diesel_migrations/migrations_internals/src/lib.rs

```diff
@@ -200,6 +200,55 @@ where
     Ok(pending)
 }
 
+pub fn revert_all_migrations_in_directory<Conn>(
+    conn: &Conn,
+    path: &Path,
+) -> Result<Vec<String>, RunMigrationsError>
+where
+    Conn: MigrationConnection,
+{
+    let number = migrations_in_directory(path)?.len() as i64;
+
+    revert_latest_migrations_in_directory(conn, path, number)
+}
+
+// TODO : see if we need a function named revert_latest_migrations
```

Comment:
As long as those functions are not part of the public API I don't think it's required to add a `revert_latest_migrations` function, otherwise this would be desirable